From b2579cc5674ee7aedefcac3ec3c710473265c3fa Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Sun, 26 Apr 2020 11:00:42 +0200
Subject: [PATCH] Remove unnecessary forward declaration

With this gone we can rename the departure types in KPublicTransport
without breaking source compatibility.
---
 src/app/livedatamanager.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/app/livedatamanager.h b/src/app/livedatamanager.h
index 902d3c9..c1b07ac 100644
--- a/src/app/livedatamanager.h
+++ b/src/app/livedatamanager.h
@@ -32,7 +32,6 @@ class TrainTrip;
 }
 
 namespace KPublicTransport {
-class Departure;
 class Manager;
 }
 
-- 
2.27.0

